<launch>
  <arg name="input" default="/rs_d435/aligned_depth_to_color/image_raw" />

  <node pkg="nodelet" type="nodelet" name="depth_distance_filter_nodelet" args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="depth_distance_filter"
        args="load aerial_robot_perception/DepthDistanceFilter depth_distance_filter_nodelet" output="screen">
    <remap from="~input" to="$(arg input)" />

    <param name="min_depth" value="0.3"/>
    <param name="max_depth" value="1.9"/>
    <param name="use_distance_from_ground" value="true"/>
    <param name="distance_from_ground" value="0.1"/>
  </node>

  <node name="opening" pkg="nodelet" type="nodelet" args="load jsk_perception/Opening depth_distance_filter_nodelet">
    <remap from="~input" to="/depth_distance_filter/output"/>
    <rosparam>
      size: 2
      iterations: 2
    </rosparam>
  </node>

</launch>
